github-git/3-fork-repo.md

```diff
@@ -0,0 +1,106 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+(fork-repository)=
+# How to Fork a GitHub Repository 
+
+
+:::{admonition} What you will learn:
+:class: tip
+
+In this lesson, you will learn how to <kbd>fork</kbd> (or create a copy of) a GitHub repo into your own **GitHub.com** account. You can practice forking the pyOpenSci example repository. 
+::: 
+
+:::{admonition} Activity: Fork a repository and modify a file
+
+**1. Fork the pyOpenSci practice GitHub repository**
+*******
+
+Fork the [pyOpenSci example repo](https://github.com/pyOpenSci/pyos-demo-package-contribute). Remember that a fork is a copy of a repository that is owned by someone else or an organization that lives in your GitHub account.
+
+**2. Open the file that you proposed changes to in the issue you selected in the [how to identify an issue lesson](2-identify-issue).**
+*******
+
+In the previous issue, you selected or identified something that you wanted to fix in a GitHub repo. Ideally, if this is your first contribution, that fix is small and could potentially be completed in the GitHub online interface. 
+
+* Select the file that you decided to work on in GitHub
+* Click on the edit <kbd><i class="fa-solid fa-pencil"></i></kbd> button in the GitHub interface.
+* Make the edits that you proposed to make.
+* Hit the <kbd>commit</kbd> button to save your edits
```

Comment:
```suggestion
```
The next lesson is about editing and commiting; this information doesn't belong here.